### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by EddyZhang38**
fix "Etcdctl: not found" issue

**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

- Fixes https://github.com/bitnami/bitnami-docker-etcd/issues/37

**Additional information**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 Trademarks: This software listing is packaged by Bitnami. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement.
 
-## TL;DR
+## Quick start
```
Comment (carrodher) [inline README.md:11]: This is a convention across READMEs for the different containers, we should maintain it
```suggestion
## TL;DR
```

**Review by carrodher — COMMENTED** [review]
Thanks for creating this PR. Please, note it is needed to sign the commits. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. You can find more info about this topic, in [the “Sign your work” section from the contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

**Comment by carrodher** [comment]
Thanks for creating this PR. Please, note it is needed to sign the commits. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. You can find more info about this topic, in [the “Sign your work” section from the contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

**Comment by EddyZhang38** [comment]
Will close this PR, since the issue is fixed in:
https://github.com/bitnami/bitnami-docker-etcd/commit/49d9dd10f9d3a4dcb51c2dcde2c8c9f8bef064b2

Thanks.

**Pull request closed without merging**